wnp-server/src/test/java/com/wonnapark/wnpserver/episode/EpisodeFixtures.java

```diff
@@ -18,29 +19,40 @@ public final class EpisodeFixtures {
     private EpisodeFixtures() {
     }
 
-    public static Webtoon createWebtoon() {
+    public static String ip() {
```

Comment:
[nit]ipv6는 생성이 안되는 것 같아서...?ㅎㅎ
```suggestion
    public static String ipv4() {
```

Reply:
wow 역시 꼼꼼하십니다,,,